x/farming/keeper/proposal_handler.go

```diff
@@ -42,6 +42,10 @@ func (k Keeper) AddPublicPlanProposal(ctx sdk.Context, proposals []*types.AddReq
 		if err != nil {
 			return err
 		}
+		terminationAcc, err := sdk.AccAddressFromBech32(p.TerminationAddress)
```

Comment:
Just a nitpick around using the same way as the right above logic. I suggest to use `p.GetTerminationAddress()` or change `p.GetFarmingPoolAddress()` to `p.FarmingPoolAddress`.

Reply:
Thank you for the suggestions, Let me modify that